source/client/output_collector_impl.cc

```diff
@@ -44,7 +44,7 @@ std::string ConsoleOutputCollectorImpl::toString() const {
 
         // The proto percentiles are ordered ascending. We write the first match to the stream.
         double last_percentile = -1.;
-        for (const double p : {.0, .5, .75, .8, .9, .95, .99, .999, 1.}) {
+        for (const double p : {.5, .75, .8, .9, .95, .99, .999, 1.}) {
```

Comment:
Is this a bug fix?

Reply:
Yes, this is a bug-fix, forgot to mention it in the PR description. 
Should I file a separate issue on this?

Reply:
It's fine, just best in future to split these out :)

Reply:
Done in https://github.com/envoyproxy/nighthawk/pull/150/commits/3836ba1cc1e39f0265e7ec02714a6de417a4fb30

Reply:
So actually, not having it introduced a bug, we would miss the minimal value. Added it back.